code/game/gamemodes/objective.dm

```diff
@@ -186,7 +186,7 @@
 
 
 /datum/objective/hijack
-	explanation_text = "Hijack the emergency shuttle by escaping alone."
+	explanation_text = "Hijack the emergency shuttle by ensuring that only non-Nanotrasen entities escape alive, aboard, and not in the shuttle brig."

```

Comment:
I really wish there was a more terse way of phrasing this, as I'm pretty sure that's why I didn't end up changing it initially.